packages/panda-preset/dist/preset.js

```diff
@@ -0,0 +1,916 @@
+"use strict";
```

Comment:
I checked in this `dist/preset.js` file as without this, `panda codegen` was failing in the `extension` with:
```
> @leather-wallet/extension@6.41.0 prepare /Users/petew/src/leather/extension
> panda codegen

🐼 error [cli] Error: Cannot find module '/Users/petew/src/leather/extension/node_modules/@cteic/panda-preset/dist/preset.js'. Please verify that the package.json has a valid "main" entry
```

I'm not sure if this is correct though, including compiled files. I am following the steps they took in this [demo](https://www.youtube.com/live/fzVyQxVvf7o?si=SulNvAaVeo-Vhk4y) but they don't actually publish the package

I published the package to my own organisation to test it a bit more:
`https://www.npmjs.com/package/@cteic/panda-preset`